assert/assertions_test.go

```diff
@@ -842,11 +829,9 @@ func TestNil(t *testing.T) {
 	if Nil(mockT, new(AssertionTesterConformingObject)) {
 		t.Error("Nil should return false: object is not nil")
 	}
-
```

Comment:
Here also your formatter committed so many changes

Reply:
oops, i'll adjust this

Reply:
removed: https://github.com/stretchr/testify/pull/1690/commits/bdff01450338f98f2c043d0c08dda4de1ddc9fda